spanner/cloud-client/src/main/java/com/example/spanner/SpannerSample.java

```diff
@@ -646,61 +662,67 @@ static void updateWithTimestamp(DatabaseClient dbClient) {
   // [START spanner_query_data_with_timestamp_column]
   static void queryMarketingBudgetWithTimestamp(DatabaseClient dbClient) {
     // Rows without an explicit value for MarketingBudget will have a MarketingBudget equal to
-    // null.
-    ResultSet resultSet =
+    // null. A try-with-resource block is used to automatically release resources held by
+    // ResultSet.
+    try (ResultSet resultSet =
```

Comment:
would it be easier to read if dbClient was on this line?

Reply:
Updated.